app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/amendment/partner/PartnerAmendContactDetailsController.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers.amendment.partner
+
+import play.api.data.Form
+import play.api.mvc._
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.actions.AuthNoEnrolmentCheckAction
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.amendment.AmendmentController
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.contact._
+import uk.gov.hmrc.plasticpackagingtax.registration.models.request.{
+  AmendmentJourneyAction,
+  JourneyRequest
+}
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.contact._
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class PartnerAmendContactDetailsController @Inject() (
```

Comment:
Elsewhere in the code the controllers tend to be named with `Amend` first? Could this follow that pattern and be called `AmendPartnerContactDetailsController`?

Reply:
Sure